File: entity/Java/Event.java
package xmu.crms.coursemanagesystem.entity;

import java.math.BigInteger;
import java.util.Date;

public class Event {

    private BigInteger id;
    private String bean;
    private String parameter;
    private Date time;

    public BigInteger getId() {
        return id;
    }

    public void setId(BigInteger id) {
        this.id = id;
    }

    public String getBean() {
        return bean;
    }

    public void setBean(String bean) {
        this.bean = bean;
    }

    public String getParameter() {
        return parameter;
    }

    public void setParameter(String parameter) {
        this.parameter = parameter;
    }

    public Date getTime() {
        return time;
    }

    public void setTime(Date time) {
        this.time = time;
    }
}

File: entity/dot net/Event.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace Xmu.Crms.Shared.Models
{
    public class Event
    {
        public long Id { get; set; }

        public string Bean { get; set; }

        public string Parameter { get; set; }

        public string Method { get; set; }

        public DateTime Time { get; set; }
    }
}